packages/govuk-frontend/src/govuk/components/input/_index.scss

```diff
@@ -118,26 +118,21 @@
   .govuk-input__prefix,
   .govuk-input__suffix {
     @include govuk-font($size: 19);
-
     box-sizing: border-box;
-    display: inline-block;
+    // Use flexbox to align text within the prefix and suffix
+    display: flex;
+    align-items: center;
```

Comment:
Works great 👍

Will it be an issue that the input still has `line-height: 1.25`?

We've got too vertical alignment methods to keep in sync now

Reply:
Good spot. From a quick play around this doesn't seem to make a difference for me. I'll do some digging and try to understand what's going on...

Reply:
Ah silly me. This is coming from `govuk-font`. I'm personally not worried about this as I'm viewing the prefixes and suffixes as purely visual aids rather than necessarily elements of the typography. I'm also not totally sure it makes a difference for the inputs since they're already driven by padding and flexbox.

Reply:
If it's not a worry that's fine. Wondered if you wanted the same approach on both?